### DISCUSSION TITLE
Use profiles for kanidm client - to manage configs for multiple servers

### DISCUSSION THREAD

**Discussion by ieugen** [kanidm/kanidm#2901] [Ideas]
hi, 

(asked on matrix first). 

regarding https://kanidm.github.io/kanidm/master/client_tools.html#kanidm-configuration . How can you use that for multiple servers? 

I am trying to setup my first kanidm deploy for a dev server and a prod server later - and I was wondering if there is a way to manage profiles. 

I think AWS cli profiles implementation is quite usable and can be a source of inspiration:
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html

**Comment by yaleman** [comment]
There's already config options for it, but not well documented. 

If you look at the file in the repo `examples/config` you can see an example of how to set up a different named instance.

**Reply by ieugen** [reply]
I guess there could be a book section for client configuration. 

The only mention I see is in the quick start. 
https://kanidm.github.io/kanidm/master/evaluation_quickstart.html?highlight=verify_ca#setup-the-client-configuration 

We can add a Client Configuration section in the book and then link it in the above location - as "Learn more about client configuration" .

**Reply by yaleman** [reply]
Raised an issue for it in #2904
